pkg/config/model.go

```diff
@@ -9,9 +9,16 @@ type GrafanaConfig struct {
 }
 
 type MimirConfig struct {
-	Address  string `yaml:"address" mapstructure:"address"`
-	TenantID string `yaml:"tenant-id" mapstructure:"tenant-id"`
-	APIKey   string `yaml:"api-key" mapstructure:"api-key"`
+	Address  string         `yaml:"address" mapstructure:"address"`
+	TenantID string         `yaml:"tenant-id" mapstructure:"tenant-id"`
+	APIKey   string         `yaml:"api-key" mapstructure:"api-key"`
+	TLS      MimirTLSConfig `yaml:"tls" mapstructure:"auth"`
```

Comment:
Is there a reason for yaml being `tls` and mapstructure being `auth`?

Reply:
I forgot to change it :) fixing